README.md

```diff
@@ -798,10 +798,37 @@ module.exports = {
 
 For production builds it's recommended to extract the CSS from your bundle being able to use parallel loading of CSS/JS resources later on.
 
-There are two possibilities to extract a style sheet from the bundle:
+There are four possibilities to extract a style sheet from the bundle:
 
-- [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin)
-- [extract-loader](https://github.com/peerigon/extract-loader) (simpler, but specialized on the css-loader's output)
+#### 1. [Asset Modules](https://webpack.js.org/guides/asset-modules/)
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  entry: [__dirname + "/src/scss/app.scss"],
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: [],
+      },
+      {
+        test: /\.scss$/,
+        exclude: /node_modules/,
+        type: "asset/resource",
+        generator: {
+          filename: "bundle.css",
+        },
+        use: ["sass-loader"],
+      },
+    ],
+  },
+};
+```
+
+#### 2. [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin)
```

Comment:
Sorry, don't see it when reviewed, let's move this to `1.` and asset mdoules to `2.`, because `mini-css-extract-plugin` is popular